hudi-aws/src/main/java/org/apache/hudi/config/GlueSyncConfig.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.config;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParametersDelegate;
+import org.apache.hudi.common.config.ConfigClassProperty;
+import org.apache.hudi.common.config.ConfigGroups;
+import org.apache.hudi.common.config.ConfigProperty;
+import org.apache.hudi.common.config.TypedProperties;
+import org.apache.hudi.hive.HiveSyncConfig;
+
+import java.util.Properties;
+
+@ConfigClassProperty(name = "Glue catalog sync based client Configurations",
+        groupName = ConfigGroups.Names.META_SYNC,
+        subGroupName = ConfigGroups.SubGroupNames.NONE,
+        description = "Configs that control Glue catalog sync based client.")
+public class GlueSyncConfig extends HiveSyncConfig {
```

Comment:
@pratyakshsharma what is the reason for renaming this class from GlueCatalogSyncClientConfig to GlueSyncConfig ?

Reply:
I had introduced GlueSyncConfig in my initial commit, and as part of resolving conflicts later, looks like I deleted the original class by mistake. Will revert the name change. 
Thank you for pointing this out.